L1Trigger/L1CaloTrigger/plugins/L1EGammaEEProducer.cc

```diff
@@ -0,0 +1,190 @@
+#include "FWCore/Framework/interface/EDProducer.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "DataFormats/L1THGCal/interface/HGCalMulticluster.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "DataFormats/L1Trigger/interface/EGamma.h"
+#include "L1Trigger/L1CaloTrigger/interface/L1EGammaEECalibrator.h"
+#include "DataFormats/Math/interface/deltaPhi.h"
+
+// we sort the clusters in pt
+bool compare_cluster_pt(const l1t::HGCalMulticluster *cl1, const l1t::HGCalMulticluster *cl2);
```

Comment:
no need for forward declaration 2 lines above the actual declaration

Reply:
done